File: week2/cpp/coordinate_geometry.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main(){
    int distance,firstX ,firstY, secondX , secondY;

    cout << "Give me the first coordinate's x" << endl;
    cin >> firstX;

    cout << "Give me the first coordinate's y" << endl;
    cin >> firstY;
        
    cout << "Give me the second coordinate's x" << endl;
    cin >> secondX;
        
    cout << "Give me the second coordinate's y" << endl;
    cin >> secondY;

    int dx = secondX - firstX;
    int dy = secondY - firstY;

    distance = sqrt((dx * dx) + (dy * dy));

    cout << "Distance is " << distance << endl;

    return 0;
}

File: lab1/python/coordinate_geometry.py
import math

print("Distance Calculator")
print("Give me first coordinate's x")
firstX = int(input())

print("Give me first coordinate's y")
firstY = int(input())

print("Give me second coordinate's x")
secondX = int(input())

print("Give me second coordinate's y")
secondY = int(input())

distance = math.sqrt(((secondX - firstX) ** 2) + ((secondY - firstY) ** 2))
print(f"The distance between two points is {distance}")
